packages/geoprocessing/scripts/dataPrep/bundleFeatures.ts

```diff
@@ -179,7 +179,7 @@ async function createGeobuf(
   // Get all features with matching ids as a single feature collection, as well
   // as the combined bbox of those features
   const { collection, extent } = await connection.one<{
-    collection: FeatureCollection;
+    collection: PrimitiveValueExpressionType;
```

Comment:
This type resolves to `any` which is less optimal than FeatureCollection.  We could add a type narrowing function to inspect and make it more specific